packages/create-bison-app/template/components/Link.tsx

```diff
@@ -46,7 +46,9 @@ export const Link = ({
       prefetch={prefetch}
       locale={locale}
     >
-      <ChakraLink {...chakraProps}>{children}</ChakraLink>
+      <ChakraLink as="span" {...chakraProps}>
+        {children}
+      </ChakraLink>
```

Comment:
`next/link` no longer requires an `<a>` child. The simplest solution here, to keep all the `next/link` props and the Chakra styling, is to just make the Chakra link a `span`.